Components/Sources/Components/Utility/HTMLUtils.swift

```diff
@@ -300,6 +303,15 @@ public struct HtmlUtils {
                 attributedString[underlineRange].underlineStyle = .single
             }
         }
+
+        // Style Strong
+        if let strongColor = styles.strongColor {
+            for strongNSRange in allStyleData.strong.completeNSRanges {
+                if let strongRange  = Range(strongNSRange, in: attributedString) {
+                    attributedString[strongRange].foregroundColor = strongColor
+                }
+            }
+        }
```

Comment:
This will only add styling to the newer AttributedString utils method, you should also add it to the `addStyling(to nsAttributedString: NSMutableAttributedString, allStyleData: AllStyleData, styles: Styles)` method in this file.